lib/td/client/api.rb

```diff
@@ -392,27 +328,22 @@ def do_get(url, params=nil, &block)
             infl.close
           end
         else
-          body = Zlib::Inflate.inflate(body)
+          begin
+            # NOTE maybe for content-encoding is msgpack.gz ?
+            body = Zlib::Inflate.inflate(body)
+          rescue
+            # NOOP
```

Comment:
This code was not assume other than the Content-Encoding is 'gzip' or 'msgpack.gz'.
However, it has been modified so as not enter other than them.

https://github.com/treasure-data/td-client-ruby/commit/ab8e45c422b99c57175cff100b9ae7d285814a38